webapp/channels/src/sass/components/_settings-modal.scss

```diff
@@ -0,0 +1,99 @@
+@charset "utf-8";
+
+.app__body {
+    .settings-modal {
+        // Tabs
+        .nav-pills {
+            > li {
+                margin-bottom: 8px;
+
+                button {
+                    height: 32px;
+                    padding: 0 12px;
+                    border-radius: 4px;
+                    color: rgba(var(--center-channel-color-rgb), 0.75);
+                    font-weight: 600;
+
+                    &:hover {
+                        background-color: rgba(var(--center-channel-color-rgb), 0.04);
+                        border-radius: 4px;
+                        color: rgba(var(--center-channel-color-rgb), 0.88);
+                    }
+                }
+
+                &.active {
+                    button {
+                        background: rgba(var(--button-bg-rgb), 0.08);
+                        color: v(button-bg);
+                    }
+                }
+            }
+        }
+
+        .modal-content {
+            border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+            background: var(--center-channel-bg);
+            border-radius: 12px;
+        }
+
+        .settings-table {
+            .settings-links {
+                width: 232px;
+                padding: 24px;
```

Comment:
what do you think about changing this to `padding: 16px`? It feels like it might be a bit too much padding right now maybe